Pipfile

```diff
@@ -42,9 +42,7 @@ graphql-core = ">=2.1,<3"
 pytz = ">=2017.2"
 six = ">=1.10.0"
 sqlalchemy = ">=1.3.0,<2"
+graphql-compiler = {editable = true, path = "."}  # necessary to make some pylint passes work
```

Comment:
Can you explain how it solves it in a comment? This statement seems to have more implications that I what I am seeing if it is able to fix pylint passes.